src/yarpmotorgui-qt/partitem.cpp

```diff
@@ -537,7 +536,7 @@ void PartItem::onSendPid(int jointIndex,Pid newPid)
 }
 void PartItem::onCalibClicked(JointItem *joint)
 {
-    const int jointIndex = joint->getJointIndex();
+    joint->getJointIndex();
```

Comment:
it seems so but it may still have side-effects, I wanted to have minimal impact...